### DISCUSSION TITLE
Twilio WhatsApp API - unable to receive media audio, photos, etc

### DISCUSSION THREAD

**Discussion by Juste-Gnimavo** [chatwoot/chatwoot#8797] [Chatwoot APIs] [answered]
Hello,
I switched from amazon to local storage. I restart the service and I 'm able to update all pictures of agents, send and received files on livechat. but we are unable to receive files sent via twilio api. but all file sent from chatwoot are received by the users.

**Comment by sojan-official** [comment, marked as answer]
ref: https://github.com/chatwoot/chatwoot/issues/8774#issuecomment-1908551102

**Reply by Juste-Gnimavo** [reply]
Thank you very much. issue solved.
